Aws/Ec2/Client.py

```diff
@@ -28,3 +29,71 @@ def describe_regions(self) -> Optional[Region]:
         :return: Region object, or None if not found
         """
         return None
+
+    def run_instances(
+            self,
+            instance_type=None,
+            image_id=None,
+            user_data=None,
+            shutdown_behavior=None,
+            network_interfaces=None,
+            min_count=1,
```

Comment:
This should not be set to any value, it is a required parameter, the application using this library has to specify them